packages/@okta/vuepress-site/.vuepress/scripts/addWorkerScript.js

```diff
@@ -23,25 +24,20 @@ parentPort.once('message', async (payload) => {
     clientManifest: JSON.parse(payload.clientManifest),
     runInNewContext: false,
     inject: false,
-    shouldPrefetch: siteConfig.shouldPrefetch || (() => false), // changed to false to prevent prefetch. site config was not overwriting this option
+    shouldPrefetch: siteConfig.shouldPrefetch || (() => false),
```

Comment:
config.js set this to 'false', but here it is () => false.   which is it?

Reply:
`config.js` did not matter what I put there, so it always falls into the `(() => false)`

Reply:
I removed prefetch from the config since its ignored it seems